### DISCUSSION TITLE
Nuxt3 support for Vuetify2

### DISCUSSION THREAD

**Discussion by kothemel** [nuxt/nuxt#17972] [Questions]
Good evening guys,

As it seems Nuxt3 does not support Vuetify2.
Are there any plans in the future to provide such support?

Thanks in advance!

**Comment by danielroe** [comment]
From https://vuetifyjs.com/en/getting-started/installation/

> The current version of Vuetify does not support Vue 3. Support for Vue 3 will come with the release of Vuetify v3.

**Comment by wobsoriano** [comment]
Vuetify 2 = Nuxt 2
Vuetify 3 = Nuxt 3

[This is how to make Vuetify 3 work with the vuetify vite plugin with Nuxt 3](https://github.com/nuxt/framework/discussions/1183#discussioncomment-1997354)
